### PR TITLE
Featuer/add support for wasm

### PR DESCRIPTION
**Pull request by hosseind2**

### CODE REVIEW

**Comment by amodm** [comment]
@hosseind2 thanks for your first contribution here! Have you tested this with the wasm being invoked inside a browser? If yes, can you share the steps to reproduce?

**Comment by hosseind2** [comment]
> @hosseind2 thanks for your first contribution here! Have you tested this with the wasm being invoked inside a browser? If yes, can you share the steps to reproduce?

welcome
actually not yet, last night I wanted to use my forked version instead of main package, but there was some errors that the path is not correct
I will try this again as soon as possible and will tell you the result

**Comment by hosseind2** [comment]
I tested it in browser and with new approach works correctly

File: Cargo.toml [hunk 1]
```diff
@@ -9,7 +9,14 @@ repository = "https://github.com/amodm/webbrowser-rs"
 readme = "README.md"
 keywords = ["webbrowser", "browser"]
 license = "MIT OR Apache-2.0"
-edition = "2015"
+edition = "2018"
+
+
+[dependencies.web-sys]
```
Comment (amodm) [inline Cargo.toml:15]: Please make this dependency conditional on wasm32

**Comment by hosseind2** [comment]
@amodm fixed

**Comment by amodm** [comment]
Lovely. Thanks. The build still fails, but I need to fix it. FYI, this will take some time to show up in the release, because I first need to fix a bunch of things on my CI.

**Comment by hosseind2** [comment]
> Lovely. Thanks. The build still fails, but I need to fix it. FYI, this will take some time to show up in the release, because I first need to fix a bunch of things on my CI.

ok thank you so much

**Comment by amodm** [comment]
@hosseind2 this is out as [0.5.5](https://crates.io/crates/webbrowser/0.5.5)

**Pull request merged**